modules/ROOT/pages/rate-limiting.adoc

```diff
@@ -0,0 +1,237 @@
+
+An accepted request passes through the API to the backend. Conversely, a rejected request displays 1429 status for HTTP (or either 400 or 500 if the API is WSDL) and does not reach the backend.
+
+A rejected request on the other hand, displays a “429 status for HTTP, ” (or either 400 or 500 if the API is WSDL) and does not reach the backend.
+
+== Examples
+
+Consider how the previously described configuration of 3 requests every 10 seconds works when the configuration is `clusterized` and `identifiers` are added.
+
+=== Configuring Identifiers by Using Regular Strings
+
+When you use the UI to add `identifiers` to the policy configuration, you can define groups of requests. The configured limits apply independently to each group. You can also use the `identifier` `#[attributes.method]` for one bucket per HTTP method in your Rate Limiting policy configuration. 
+
+//If you can split the universe of possible requests into a finite number of groups, you can assign each group with a selector key (String). One Rate Limit algorithm is created per key.  
+
+An `identifier` is a non-obligatory parameter.  By default, the `identifier` has no value.
+Based on whether you accept the default or provide a value, the policy performs in the following ways:
+
+* Not configured (default)
++
+The Rate Limiting quota applies to every request per bucket or group.
+* Configured for an obligatory HTTP header
++
+Each header has its own quota. Header values are case-sensitive. Quotas are created using the _lazy creation_ strategy.
+* Configured for a non-obligatory HTTP header
++
+Custom header, payload, query parameter, or expression values each have their own quotas.  
++
+The `identifier`, if not sent in the request, defaults to an empty value, having its own quota. This behavior allows the Rate Limiting policy to be applied to an API consumed by uncontrolled clients, and at the same time accommodates special buckets for the clients sending the `identifier` (which might be internal controlled services).
+//ask Mariano what does the braces refer to? What might be internal controlled services
+
+The following example shows the order of events that occur over a period of time using the `identifier` `#[attributes.method]` for a limit of 3 requests every 10 seconds:
+
+image::rate-limiting-configure-identifiers-example.png[Rate Limiting Configure Identifiers Example,70%,85%]
+In the example:
+
+* Every HTTP method is allowed 3 requests every 10 seconds (in this example, only GET and POST requests are made to the API).
+* The Rate Limiting policy works in a fixed-window fashion.
++
+For more information, see the window size bracket.
+//Mariano--See what window size bracket, and how does this explain "fixed-window fashion"?
```

Comment:
You need to look at the picture to see the bracket. The "fixed-window fashion" refers to that bracket, one that starts and closes at well defined points in time.

Reply:
Done! Thanks!